frontend/src/components/maps/leaflet/PointClusterer.tsx

```diff
@@ -321,23 +312,7 @@ export const PointClusterer: React.FC<PointClustererProps> = ({
                 selected.propertyTypeId === PropertyTypes.PARCEL &&
                   dispatch(storeParcelDetail(null));
               }}
-            >
-              <Popup autoPan={false}>
-                <PopupView
-                  propertyTypeId={selected.propertyTypeId}
-                  propertyDetail={selected.parcelDetail}
-                  zoomTo={() =>
-                    leaflet.map?.flyTo(
-                      [
-                        selected.parcelDetail!.latitude as number,
-                        selected.parcelDetail!.longitude as number,
-                      ],
-                      MAX_ZOOM,
-                    )
-                  }
-                />
-              </Popup>
-            </SelectedPropertyMarker>
+            />
```

Comment:
From what I can tell, a SelectedPropertyMarker appears after someone clicks "View" from the property list view, in which case the property detail view is open anyway

Reply:
SelectedPropertyMarkers show up whenever the sidebar is open in view or update mode.